src/App.js

```diff
@@ -1,19 +1,32 @@
 /* 
-  INCODE WEB SDK AND REACT INTEGRATION 
+INCODE WEB SDK AND REACT INTEGRATION 
 
-  MORE INFO:
-  https://developer.incode.com/docs/tutorial-creating-an-identity-validation-app
+MORE INFO:
+https://developer.incode.com/docs/tutorial-creating-an-identity-validation-app
 */
 
 import React, { useEffect, useState, useRef } from "react";
-import { getIncodeToken } from "./incode-session-service";
-import translations from "./translations";
+import { incode } from "./incode";
 import "./App.css";
-let incode;
+
 let incodeSession;
-let SDKActive = false;
 let container;
 
+//Function to fetch the onboarding session from your backend
+async function startOnboardingSession() {
```

Comment:
smaller function that calls the backend instead of trying to create the session here